clang/test/SemaSYCL/Inputs/sycl.hpp

```diff
@@ -143,6 +143,7 @@ class __attribute__((sycl_special_class)) __SYCL_TYPE(accessor) accessor {
   using PtrType = typename DeviceValueType<dataT, accessTarget>::type *;
   void __init(PtrType Ptr, range<dimensions> AccessRange,
               range<dimensions> MemRange, id<dimensions> Offset) {}
+  void __init_esimd(PtrType Ptr) {}
```

Comment:
It probably makes sense to add this together with corresponding test in a separate change. Otherwise people looking at the history will be confused by the fact that change named `Support multiple call operators in kernel functor` for some reason adds `__init_esimd` to the mock accessor and then doesn't do anything about it.